### DISCUSSION TITLE
Budgets - Why does the old value show up?

### DISCUSSION THREAD

**Discussion by bergernetch** [firefly-iii/firefly-iii#5597] [Firefly III] [answered]
Something seems wrong to me, or I do not understand how it is intended to use, even after reading the documentation.

To replicate (I tried it on the demo 5.6.10 as well as my own installation 5.6.10):
Budgets > new Budget
set "Auto-budget amount" to e.g. 999
store it

The Budget shows up in the list.
There is an input field in the list, column "Budgeted", showing 999.00
When I change this amount directly, say to 777, it will be stored and the "Left" column updated.

Then click on the Change Icon left (Pencil) of this entry, the "Auto-budget amount" shows 999 again!
Why?

**Comment by JC5** [comment, marked as answer]
The auto-budget doesn't overrule your changes. It will set the amount automatically for the start of next month.

Each budget, for each month can have individual amounts, you can change them as you see fit. The changes you make on `/budgets` only apply to the current period.

**Comment by bergernetch** [comment]
Thank you. I see. I am extra confused, because we had the same problem as https://github.com/firefly-iii/firefly-iii/issues/4960 - Most of our created budgets show up in the list with 0 in the "Budgeted" column and completely empty fields in "Spend" and "Left".
When creating new ones now, it does not happen anymore (I do not know which version I was on when we created to problematic budgets, but now I am on 5.6.10)

**Comment by bergernetch** [comment]
One more question: When I navigate to - say June - all budgets show 0 except the yearly ones. Should not the "Auto-budget amount" be set for every month, if I do not overwrite it?

**Reply by JC5** [reply]
Yes, but they won't be created in advance. For June, expect them on the first of June, unless you've set an amount yourself first.

**Comment by bergernetch** [comment]
Or how can one remove a handmade change for a time period? Remove the record from budget_limits table in db? There seems to be no way in the UI.
The whole thing is still confusing for me, sorry.

**Reply by JC5** [reply]
There's an arrow next to an amount. Press it.

**Reply by bergernetch** [reply]
For many entries (with an amount of 0) there is no arrow. But I found out that I can press the + sign and set a value for my existing currency. After that the arrow shows up and I can remove it.

**Reply by JC5** [reply]
The 0 is just the default when no value is set. They will show up again.
